### ISSUE TITLE
Update Screenshots

### ISSUE THREAD

**Issue by Xerus2000** [sghpjuikit/player#25]
Some Screenshots are outdated, and they all need a somewhat better description to increase presentability.

**Comment by sghpjuikit** [comment]
I agree. In addition the readme in general needs to be more concise and on point, more helpful and complete. This will take an effort and I may not have enough time. Objection noted, I'll see what I can do. Of course, if someone will beat me to it, I won't complain, but yes, it's been on my to do list already.

**Comment by omachala** [comment]
Hi! I can help update these screenshots.

I maintain [heroshot](https://heroshot.sh), a screenshot automation tool that captures screenshots from a config file using Playwright. Once set up, regenerating screenshots becomes a one-command operation.

Happy to help if this would be useful!
